src/Mvc/Mvc.ApiExplorer/src/ApiResponseTypeProvider.cs

```diff
@@ -146,7 +146,7 @@ private static List<IApiResponseMetadataProvider> GetResponseMetadataAttributes(
                 // and respond to the incoming request.
                 // Querying IApiResponseTypeMetadataProvider.GetSupportedContentTypes with "null" should retrieve all supported
                 // content types that each formatter may respond in.
-                contentTypes.Add((string)null);
+                contentTypes.Add((string)null!);
```

Comment:
What's going on here?

Reply:
It's the same weird thing as https://github.com/dotnet/aspnetcore/pull/33059#discussion_r640314837. When used in runtime, the usual code paths will throw if you supply a null value (https://github.com/dotnet/aspnetcore/blob/main/src/Mvc/Mvc.Core/src/ProducesAttribute.cs#L40-L43, https://github.com/dotnet/aspnetcore/blob/main/src/Mvc/Mvc.Core/src/ProducesAttribute.cs#L120). But in API explorer, we want to have it do some questionable things and run a for loop over at least once, so it hacks around it.